src/test/java/com/epam/tam/pages/NewHCPAccountPage.java

```diff
@@ -0,0 +1,93 @@
+package com.epam.tam.pages;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Select;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import com.epam.tam.exception.ElementByXpathNotFoundException;
+import com.epam.tam.exception.ValueNotFoundInDropDownException;
+import com.epam.tam.util.Property;
+
+public class NewHCPAccountPage extends Page {
+
+	final static Logger logger = LogManager.getLogger(NewHCPAccountPage.class);
+
+	@FindBy(xpath = "//div[@class='modal-content' and .//*[contains(normalize-space(h1), 'New User')]]")
+	private WebElement newHCPAccountForm;
+
+	@FindBy(id = "name")
+	private WebElement nameField;
+
+	@FindBy(xpath = "//*[@formcontrolname='lastName']")
+	private WebElement lastNameField;
+
+	@FindBy(id = "email")
+	private WebElement emailField;
+
+	@FindBy(id = "sites")
+	private WebElement sitesDropDown;
+
+	@FindBy(id = "org")
+	private WebElement orgAdminCheckbox;
+
+	@FindBy(id = "site")
+	private WebElement siteAdminCheckbox;
+
+	@FindBy(xpath = "//button[@type='submit']")
+	private WebElement saveButton;
+
+	public NewHCPAccountPage(WebDriver driver) {
+		super(driver);
+	}
+
+	public boolean newHCPAccountFormDisplayed() {
+		waitForVisibility(newHCPAccountForm);
+		return newHCPAccountForm.isDisplayed();
+	}
+
+	public AdminPage createNewHCPAccount(String name, String lastName,
+			String email, String site, boolean orgAdmin, boolean siteAdmin)
+			throws ValueNotFoundInDropDownException, ElementByXpathNotFoundException {
+		logger.info(
+				"Creating account with name {}, lastName {}, email {}, site {}, orgAdmin {}, siteAdmin {}",
+				name, lastName, email, site, orgAdmin, siteAdmin);
+		nameField.sendKeys(name);
+		lastNameField.sendKeys(lastName);
+		emailField.sendKeys(email);
+		selectValueInDropDown(sitesDropDown, site);
+		if(orgAdmin) {
+			orgAdminCheckbox.click();
+		}
+		if (siteAdmin) {
```

Comment:
There are two checkboxes, they may be both selected, so 'if' is appropriate here.